admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -11,7 +11,7 @@
 
 <header>
   <nav class="main-header">
-
+    <div id="ajax_spinner" class="loading-spinner ajax_spinner"><div class="dot1"></div><div class="dot2"></div><div class="dot3"></div></div>
```

Comment:
If the spinner is a standard component, can it be put in a separate template and then included in this one?